hera_mc/correlator.py

```diff
@@ -913,6 +927,10 @@ def create(cls, time, antenna_number, antenna_feed_pol, snap_hostname,
         if antenna_feed_pol not in ['e', 'n']:
             raise ValueError('antenna_feed_pol must be "e" or "n".')
 
+        if (fem_switch is not None
+                and fem_switch not in ['antenna', 'load', 'noise']):
```

Comment:
since corr_cm runs in python 3 there _shouldn't_ be any bytes vs string issue with this. I'll report back after maintenance day.